custom/workflow-helper/postgresql-helper/load-script/cmdCommand.py

```diff
@@ -0,0 +1,64 @@
+import flask 
+import psycopg2 
+import time, random
+import secrets
+import string, os
+import base64
+
+from kubernetes import client, config
+if os.getenv('KUBERNETES_SERVICE_HOST'):
+    configs = config.load_incluster_config()
+else:
+    configs = config.load_kube_config()
+v1 = client.CoreV1Api()
+try:
+    secret = v1.read_namespaced_secret("postgres-application.credentials", "litmus")
```

Comment:
make ns as tunable and secret name as a global variable or const

Reply:
Thank you @ispeakc0de, I'm updating it